src/ex1b.ml

```diff
@@ -0,0 +1,17 @@
+let () =
+  let doc = Dom_html.document in
+  let div = Dom_html.createDiv doc in
+  let h2 = Dom_html.createH2 doc in
+  let button = Dom_html.createButton ~_type:(Js.string "button") doc in
```

Comment:
Using `Tyxml_js` here would make things much nicer and would remove nearly all dom manipulation.